src/chainstate/stacks/boot/pox.clar

```diff
@@ -250,36 +229,30 @@
                                                             (num-cycles uint)
                                                             (amount-ustx uint)
                                                             (i uint))))
-    (let (
-        (reward-cycle (+ (get first-reward-cycle params) (get i params)))
-        (i (get i params))
-    )
+    (let ((reward-cycle (+ (get first-reward-cycle params) (get i params)))
```

Comment:
It's hard to see what's changed here with all the formatting-related changes.  Can you provide a summary?

Reply:
Nothing ended up being changed here -- it was all formatting. It's just adding whitespace so that everything in the same context is at the same tab depth (i.e., everything in the same let block should be at the same tab depth).